src/main/resources/migration/V1__Create_tables_And_Insert.sql

```diff
@@ -0,0 +1,19 @@
+
+-- ----------------------------
+-- Table structure for user_info
+-- ----------------------------
+DROP TABLE IF EXISTS user_info;
+CREATE TABLE user_info (
```

Comment:
PR1: First, what scheme you put this table in? Second, why do you think 32 length is enough for username, role and "describe"? And describe here is a verb, but all other columns are nouns. Same question for 128 length of the password.